war/src/main/js/components/dropdowns/templates.js

```diff
@@ -47,7 +47,7 @@ function menuItem(options) {
   const tag = itemOptions.type === "link" ? "a" : "button";
 
   const item = createElementFromHtml(`
-      <${tag} class="jenkins-dropdown__item" href="${itemOptions.url}">
+      <${tag} class="jenkins-dropdown__item ${itemOptions.clazz || ""}" ${itemOptions.url ? `href="${itemOptions.url}"` : ""} ${itemOptions.id ? `id="${itemOptions.id}"` : ""}>
```

Comment:
Should be `xmlEscape`'d (probably including the URL even though it's not new here).